### PR TITLE
fix: Use Job multiple input URLs to segment videos

### PR DESCRIPTION
**Pull request by shantanutomarr**
- If we only want to segment videos, without transcoding them, we can now-
  1. In Job Create API, pass `inputs` with data for each Output.
      - Eg: `inputs = [{"url": "360p_input.mp4", "name": "360p"}, {"url": "720p_input.mp4", "name": "720p"}]`
  2. And in settings pass `{"segment_only": true}`

### CODE REVIEW

File: lumberjack/apps/jobs/models.py [hunk 1]
```diff
@@ -78,6 +78,32 @@ def update_video_duration(self):
         async_update_video_duration_in_job.delay(self.id)
 
     def create_outputs(self):
+        if self.settings.get("segment_only") and self.input_urls:
+            self._create_outputs_for_segment_only_job()
```
Comment (shantanutomarr) [inline lumberjack/apps/jobs/models.py:82]: Currently `_create_outputs_for_segment_only_job ` will not accept Transcoding job. We can adapt it to do so in another commit.

**Review by bharathwaaj — APPROVED** [review]
